samples/ios/app/glean-sample-app/AppDelegate.swift

```diff
@@ -39,6 +38,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
+    func application(_: UIApplication,
+                     open url: URL,
+                     options _: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        let host = url.host?.removingPercentEncoding
```

Comment:
nit: what about moving these checks within the `handleCustomUrl` so that the app would only be required to do a function call?

Reply:
I thought about that, and can do that, it just felt a little weird if the app was doing something for itself as it would have a 'host' that would be specific to it.  It would basically be a no-op to call it if the 'host' wasn't "glean", so let me make that change.  I still have tests to add as well...